charactersheet/charactersheet/viewmodels/character/armor.js

```diff
@@ -15,13 +15,7 @@ function ArmorViewModel() {
         'armorName asc': { field: 'armorName', direction: 'asc'},
         'armorName desc': { field: 'armorName', direction: 'desc'},
         'armorType asc': { field: 'armorType', direction: 'asc'},
-        'armorType desc': { field: 'armorType', direction: 'desc'},
-        'armorDexBonus asc': { field: 'armorDexBonus', direction: 'asc'},
-        'armorDexBonus desc': { field: 'armorDexBonus', direction: 'desc'},
```

Comment:
Out of curiosity, why did you remove the sorting from ```armorDexBonus```? The column sort is still bound in the table header.